src/sentry/api/base.py

```diff
@@ -192,6 +193,8 @@ def _create_api_access_log(self):
                 path=self.request.path,
                 caller_ip=self.request.META.get("REMOTE_ADDR"),
                 user_agent=self.request.META.get("HTTP_USER_AGENT"),
+                db_queries=len(connection.queries),
+                db_query_time=sum(float(query["time"]) for query in connection.queries),
             )
             api_access_logger.info("api.access", extra=log_metrics)
         except Exception:
```

Comment:
I wonder if it would be worth adding a "if in development, actually throw an Exception" so that way we can catch exceptions in development while keeping prod safe.